src/main/java/com/datastax/loader/CqlDelimParser.java

```diff
@@ -157,12 +130,16 @@ private void processCqlSchema(Session session) throws ParseException {
 
 
     private List<SchemaBits> schemaBits(String in, Session session) throws ParseException {
-        KeyspaceMetadata km = session.getCluster().getMetadata().getKeyspace(keyspace);
+        keyspace = keyspace.replace("\"","");
+        System.out.println(keyspace);
+        KeyspaceMetadata km = session.getCluster().getMetadata().getKeyspace("\""+keyspace+"\"");
+        System.out.println(keyspace);
```

Comment:
These sysouts look out of place, perhaps tracing that was intended to be removed after debugging?